### ISSUE TITLE
NVDA no longer gains focus properly when opening VS Code

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#2003]
Issue Type: <b>Bug</b>

/CC -isidorn

NVDA is completely unable to focus on the control with keyboard focus in VS code when alt tabbing into it from another application. This happens 100% of the time for me.

This is very similar to -28316. When this occurs, Vs Code is completely unusable with NVDA (pressing tab and arrow keys doesn't read anything.

I am completely unable to use Vs Code. Only started happening today when I got a new insiders build. Since I haven't updated NVDA for a couple of months, a recent change in Code is a likely suspect.

VS Code version: Code - Insiders 1.69.0-insider (0b74e15affc1b44a5b7cd671dd0a02d3ed6d51a7, 2022-06-15T05:16:09.187Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 5.10.102.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 PRO 4750U with Radeon Graphics (16 x 1697)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.23GB (15.97GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+Ubuntu/home/dickson/source/backend-api --remote=wsl+Ubuntu --crash-reporter-id 1304d64e-abdb-456c-beb6-72c33120f731|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.10.102.1-microsoft-standard-WSL2|
|CPUs|AMD Ryzen 7 PRO 4750U with Radeon Graphics (16 x 1696)|
|Memory (System)|24.44GB (18.99GB free)|
|VM|0%|
</details><details><summary>Extensions (26)</summary>

Extension|Author (truncated)|Version
---|---|---
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.240.0
remote-ssh|ms-|0.82.1
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
vscode-remote-extensionpack|ms-|0.21.0
vscode-eslint|dba|2.2.2
dendron|den|0.99.0
dendron-markdown-links|den|0.6.20
dendron-markdown-preview-enhanced|den|0.10.57
dendron-markdown-shortcuts|den|0.12.1
dendron-paste-image|den|1.1.0
dendron-snippet-maker|den|0.1.6
gitlens|eam|12.1.0
prettier-vscode|esb|9.5.0
copilot|Git|1.29.6152
rest-client|hum|0.24.6
rainbow-csv|mec|2.4.0
python|ms-|2022.8.0
vscode-pylance|ms-|2022.6.20
jupyter|ms-|2022.5.1001601848
vsliveshare|ms-|1.0.5625
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
vscode-mermaid-editor|tom|0.14.0
vscode-remark|uni|1.3.0

</details>
